src/encode_task_macs2_chip.py

```diff
@@ -86,7 +79,7 @@ def macs2(ta, ctl_ta, chrsz, gensz, pval_thresh, shift, fraglen, cap_num_peak,
     run_shell_cmd(cmd0)
 
     cmd1 = 'LC_COLLATE=C sort -k 8gr,8gr "{}"_peaks.narrowPeak | '
-    cmd1 += 'awk \'BEGIN{{OFS="\\t"}}{{$4="Peak_"NR; if ($2<0) $2=0; if ($3<0) $3=0; print $0}}\' > {}'
+    cmd1 += 'awk \'BEGIN{{OFS="\\t"}}{{$4="Peak_"NR; if ($2<0) $2=0; if ($3<0) $3=0; if ($10==-1) $10=$2+int(($3-$2+1)/2.0); print $0}}\' > {}'
```

Comment:
This line is long and cryptic. Could you a) turn it into a multiline string and b) add some comment in the code of what it does?
```suggestion
    cmd1 += (
        'awk \'BEGIN{{OFS="\\t"}}{{$4="Peak_"NR; if ($2<0) $2=0; if ($3<0) $3=0; if ($10==-1) ' 
        '$10=$2+int(($3-$2+1)/2.0); print $0}}\' > {}'
    )
```

Reply:
thanks, fixed.